src/scraper/scrape.py

```diff
@@ -69,9 +73,11 @@ def main(feed_id, config_path):
         content = scrape(url)
         now = time.time()
         file_path = f'{config["feeds"][0]["feed_name"]}/{now}.bin'
```

Comment:
why is file extension .bin?

Reply:
also think might be better to specify the format of the time (e.g. YYYY/MM/DD/HH:MM:dd

Reply:
I just randomly choose it as it’s binary.
What’s your preferred file extension?
I thought about having none but we may want some way to distinguish files living in the same bucket.

Reply:
I also used feed name instead of feed ID in the parent bucket name since mobility DB may not have some feeds and we don’t want user given dummy feed id clashed into those

Reply:
Can you use `binpb` as recommended [here](https://protobuf.dev/programming-guides/techniques/#suffixes)?  Thanks!